packages/core/schematics/migrations.json

```
@@ -1,3 +1,9 @@
 {
-  "schematics": {}
+  "schematics": {
+    "standalone-false": {
```

I think it'd be great to rename this migration to something like this:
```suggestion
    "explicit-standalone-flag": {
```